cuckoo/table.go

```diff
@@ -38,19 +38,29 @@ func (i *Item) Equals(other *Item) bool {
 	}
 	return i.Bucket1 == other.Bucket1 &&
 		i.Bucket2 == other.Bucket2 &&
-		i.Id == other.Id;
+		i.Id == other.Id
+}
+
+// Bucket returns the bucket in a table that the Item is in, if it is in the table.
+func (i *Item) Bucket(table *Table) int {
```

Comment:
Looks like this is unused? 
I'd remove this if unused. If used, seems more appropriate as a table.Find() => returns bucket number or something like that.